src/mods.cpp

```diff
@@ -80,12 +80,15 @@ void parseModContents(ModSpec &spec)
 	}
 }
 
-std::map<std::string, ModSpec> getModsInPath(std::string path, bool part_of_modpack)
+std::map<std::string, ModSpec> getModsInPath(std::string path, bool part_of_modpack, std::string worldpath)
```

Comment:
The change is for efficiency.  Instead of creating a new `std::string` object and copying the data in it you just pass a pointer.